### ISSUE TITLE
email error results in completed record in database

### ISSUE THREAD

**Issue by katherinepdavis** [BlueprintKansas/ksvotes.org#280]
If I force an email failure, I still see a "ab_completed_at" value for that record. I believe that means the db doesn't record email failures differently from successes.

**Comment by karpet** [comment]
good catch. The "completed_at" value gets set whenever the Mailer returns ok, and since we now catch errors and handle them gracefully, the Mailer is always going to return ok. I will make the "completed_at" logic more specific.

**Comment by karpet** [comment]
fixed in #289
